packages-next/fields/src/types/image/Implementation.ts

```diff
@@ -0,0 +1,235 @@
+import { PrismaFieldAdapter, PrismaListAdapter } from '@keystone-next/adapter-prisma-legacy';
+import { ImageData, KeystoneContext, BaseKeystoneList } from '@keystone-next/types';
+import { Implementation } from '../../Implementation';
+import { handleImageData } from './handle-image-input';
+
+const SUPPORTED_IMAGE_EXTENSIONS = ['jpeg', 'png', 'webp', 'gif'];
```

Comment:
We have this list in here and in the `create-image-context` file, in the future when fields is part of keystone, we should probably have a single reference to this stored in a constants file of some variety.